content/guides/cli/scripts/manage-groups-collaborations.md

```diff
@@ -0,0 +1,193 @@
+---
+rank: 7
+related_endpoints: []
+related_guides:
+  - authentication/oauth2
+  - cli/scripts/provision-users-folders
+  - cli/scripts/deprovision-users
+  - cli/scripts/user-zones-mass-update
+  - cli/scripts/report-inactive-users
+related_pages:
+  - sdks-and-tools
+required_guides:
+  - authentication/oauth2/oauth2-setup
+related_resources: []
+
+
+---
+# Manage groups and collaborations
+
+<!-- markdownlint-disable line-length -->
+
+## Script structure
+
+This script uses Box CLI to create or update groups, add users to them, and finally to create collaborations between groups and folders.
+The script consists of the following two parts. You can run them both or select one of them.
+
+### Create or update groups 
+ 
+1. The script uses the `.csv` file you specify for the `UserGroupAdditionPath`
+   parameter. The file lists group names and user emails. When creating the file, you can use the same group name for several users, and assign one user to several groups.
+   For example:
+
+   |`GroupName`| `UserEmail`|
+   |-----------|------------|
+   |Group1| ManagedUser1@test.com|
+   |Group1| ManagedUser2@test.com|
+   |Group2| ManagedUser3@test.com|
+   |Group3| ManagedUser1@test.com|
+
+2. If the group doesn't exist, the script creates it. If it does exist, the script can update the entries based on the provided data.
+
+### Create or update collaborations
+
+1. The script uses the `.csv` file you specify for the
+   `CollaborationsCreationPath` parameter. The file lists group names, folder IDs, and collaboration roles. 
+2. For each row, the script checks if a group exists and if it's not already
```

Comment:
Also, if group already exists, but CollaborationRole changed, script will update it if the parm `-UpdateExistingCollabs ` is passed

From the readme
> There may be cases where a group has already been a collaborator for a specific folder, but with a role other than that defined in the CSV file. This script will simply inform you about that case without making any change to existing collaboration. But if you want to update an existing collaboration with role defined in CSV file, you need to pass an additional switch -UpdateExistingCollabs when running the script.

Reply:
Added this bit to the explanation.